scripts/hashes.json

```diff
@@ -0,0 +1,142 @@
+  "oracles/REthPriceOracle.sol": "5766fd7faad8b5944cf4d80caa989c8c8fc9b2768985ac15dc7d2a0335ce5041",
+  "oracles/RgtTempPriceOracle.sol": "32c9e3292a3b24be897ab4c91dc1e0b23b9cf71a4030bb90cb5f608cbf1341c8",
+  "oracles/SaddleLpTokenPriceOracle.sol": "89dff1c5484fed295fd4689eca01c7c309493e7b0a2febe41cc1d6f441897fd6",
+  "oracles/StakedFodlPriceOracle.sol": "dd5c40a9e410ae2c5e6ccee1c93fcde676c7931215a5fa8c8a4c0e765847719f",
+  "oracles/StakedSdtPriceOracle.sol": "ac86498ae9c416a77e065df9481745f98dd154b53c3e2c571e62f7a2a55d1217",
+  "oracles/StakedSpellPriceOracle.sol": "b8f18a04c0a3c271561a37ef9d40325bc7e46afed6b5093fe1d98ff2d4da8a35",
+  "oracles/SushiBarPriceOracle.sol": "bc13a7547fbc3ea0ffe5e636e1d4cc9aa99c4c9a36851e1677de6c29a819849c",
+  "oracles/SynthetixPriceOracle.sol": "53d8a898947fad454daec96a40cbf9c1d328ddeec9bd5ab0c2dcc814a6756b7c",
+  "oracles/TemplePriceOracle.sol": "653adf41e1b02b403634e7a6d31924b56df8230d2b0152d92c372dbc2874fe3e",
+  "oracles/TokemakPoolTAssetPriceOracle.sol": "9259fb85ebacd67ef5ad75f5e57d787391ab48cf2875ab7e3c284c579c8f22cf",
+  "oracles/UniswapLpTokenPriceOracle.sol": "ada07a8e36a7db225c627777c7d367bfd2d13e21d138762dfaa99bc9f3bd76f2",
+  "oracles/UniswapTwapPriceOracle.sol": "0ff46c3dd1eb4e9c25a7a4ab0a7ee36a2a14a1ef76145b12eb1003ece40ba591",
+  "oracles/UniswapTwapPriceOracleRoot.sol": "21ff0613099b7baaae211e0fd738f59bb46c92e6e1b435dc784f65ace263b6e5",
+  "oracles/UniswapTwapPriceOracleV2.sol": "67efc9872fc88a2d1afae77d583dd2ce81ef06b0d0f4c11e4a484335d27f83d6",
+  "oracles/UniswapTwapPriceOracleV2Factory.sol": "82872ee1a254d907feb3dc5bc54f6fa720fb37fb29941e111b388135fc83844f",
+  "oracles/UniswapTwapPriceOracleV2Root.sol": "37435c3c1e2814d2e9d4523b901db7d4d09ff5dd11cfbdcfb494cd273db36046",
+  "oracles/UniswapV3TwapPriceOracle.sol": "2767f71b8ac2dae65c53f04fb52ebd16849e0871ef6b1c5df116c53c9286d5c0",
+  "oracles/UniswapV3TwapPriceOracleV2.sol": "01527043b5dfce6b3ec7af49b2f7e7d198510c38319975037b3ad57722bbb318",
+  "oracles/UniswapV3TwapPriceOracleV2Factory.sol": "5eb38a86fd409a618a3f78adf1cd94da3f12c76a4069207bc0efefa2334261ab",
+  "oracles/VoltPriceOracle.sol": "9fcff1308a8f87de93d7cc84bf3a3f47c7f1051c42438035817503675d26d6a5",
+  "oracles/WSSquidPriceOracle.sol": "e27382c3fa2c5d31fd5c83b89ff49db4fbb92b4bc2333f9b69b8453ea9cc7a41",
+  "oracles/WSTEthPriceOracle.sol": "dbaf2d9c0fe557dcae3d5f46801757032f65e4cf41a51dd0b0b7651c15669ea2",
+  "oracles/WXBtrflyPriceOracle.sol": "d75319f914bfb6d762185f3ebc38263dee2a5390c05148011c5c372b8ed497e0",
+  "oracles/XVaultPriceOracle.sol": "0af7510c5423e955cf3e5120c74db1a069294f528c4f38ce58d3c883062ad079",
+  "oracles/YVaultV1PriceOracle.sol": "a14db0885769ac440ff253c56374ca1a77915b0265508a92b248444370948fcd",
+  "oracles/YVaultV2PriceOracle.sol": "ee7bf1eb0bc53e3bfd4807def3638f0270be78d8f3cd9d5e008f15247dd2cc59",
+  "utils/InitializableClones.sol": "cffec4fdbeec405808e73937685c37862b553ebe1cd3c69e9305774a68a5a1ea"
```

Comment:
I've now added the filtering by maintaining a SHA256 file hash list. I can imagine that in the future we might want to add a `data` directory in which we keep these kinds of files. For now I think it is OK.